.github/workflows/build_shadow.yml

```diff
@@ -0,0 +1,24 @@
+# Syntax reference:
+# https://help.github.com/en/actions/automating-your-workflow-with-github-actions/workflow-syntax-for-github-actions
+
+name: Test
+on: [push, pull_request]
+
+jobs:
+  testing:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
```

Comment:
Super nit-pick: Checkout is capitalized, but the other steps are not.

Reply:
Done. Agreed it should be consistent. I could go either way but arbitrarily capitalized them all since that's what the examples do.